interface/package.json

```diff
@@ -14,7 +14,9 @@
   "engines": {
     "node": ">=4.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "hclusterjs": "^1.3.0"
```

Comment:
Okay, sounds good. Keep in mind that the `vendors/` and `node_modules/` directories exist only for the build and are not used by the deployed code. The minifying step used for the final "packaging" of the files will merge this file along with the others pulled from `vendors/` into our own customize `adage-ver#.js` file (currently `interface/bin/assets/adage-0.2.1a.js`) so, in the end, this file does actually get treated the same as the others in `vendors/`.